### ISSUE TITLE
Loss not changing when training with Kaggle dataset

### ISSUE THREAD

**Issue by Hamdi-Ben-Abdallah** [Utkarsh-Deshmukh/Blurry-Image-Detector#1]
Hello,

Thank you for this beautiful work

I have not changed the settings and am using your code as it is with the Kaggle dataset but I have the loss that does not change when training.

Why don't I get the same results as you?
Thank you in advance.

Epoch 0 | Loss   0.45
Epoch 1 | Loss   0.42
Epoch 2 | Loss   0.42
Epoch 3 | Loss   0.42
Epoch 4 | Loss   0.41
Epoch 5 | Loss   0.41
Epoch 6 | Loss   0.41
Epoch 7 | Loss   0.41
Epoch 8 | Loss   0.41
Epoch 9 | Loss   0.41
Epoch 10 | Loss   0.41
Epoch 11 | Loss   0.41
Epoch 12 | Loss   0.41
Epoch 13 | Loss   0.41
Epoch 14 | Loss   0.41
Epoch 15 | Loss   0.41
Epoch 16 | Loss   0.41
Epoch 17 | Loss   0.41
Epoch 18 | Loss   0.41
Epoch 19 | Loss   0.41
Epoch 20 | Loss   0.41
Epoch 21 | Loss   0.41
Epoch 22 | Loss   0.41
Epoch 23 | Loss   0.41
Epoch 24 | Loss   0.41
Epoch 25 | Loss   0.41
Epoch 26 | Loss   0.41
Epoch 27 | Loss   0.41
Epoch 28 | Loss   0.41
Epoch 29 | Loss   0.41
Epoch 30 | Loss   0.41
Epoch 31 | Loss   0.41
Epoch 32 | Loss   0.41
Epoch 33 | Loss   0.41
Epoch 34 | Loss   0.41
Epoch 35 | Loss   0.41
Epoch 36 | Loss   0.41
Epoch 37 | Loss   0.41
Epoch 38 | Loss   0.41
Epoch 39 | Loss   0.41
Epoch 40 | Loss   0.41
Epoch 41 | Loss   0.41
Epoch 42 | Loss   0.41
Epoch 43 | Loss   0.41
Epoch 44 | Loss   0.41
Epoch 45 | Loss   0.41
Epoch 46 | Loss   0.41
Epoch 47 | Loss   0.41
Epoch 48 | Loss   0.41
Epoch 49 | Loss   0.41
Train Accuracy =  0.8181365664610407

**Comment by Utkarsh-Deshmukh** [comment]
can you try re-training once only with de-focus blurred and sharp images (not using the motion blur images)
If you see that the training accuracy is high, then you can think of introducing motion blur images to the training.

**Comment by Hamdi-Ben-Abdallah** [comment]
I re-trained the model with blurred and sharp images without using motion blur images but the accuracy of the training is very low.
I have an accuracy of about 40% with the same setup as you. The only change I made is :
batch_size = 32
num_epochs = 1000

**Comment by Utkarsh-Deshmukh** [comment]
can you try to run the pre-trained model on the test data using the script `Test_main.py` and see what kind of performance you get?
